src/facilito/models/download.py

```diff
@@ -10,27 +10,36 @@ class YoutubeDLLogger:
 
     def debug(self, msg):
         """Debug message"""
-        print(msg)
+        #print(msg)
+        pass
 
     def info(self, msg):
         """Info message"""
-        print(msg)
+        #print(msg)
+        pass
 
     def warning(self, msg):
         """Warning message"""
-        print(msg)
+        #print(msg)
+        pass
 
     def error(self, msg):
         """Error message"""
         raise DownloadError(f"Downloading video failed. {msg}")
 
     # TODO: implement custom parser for progress bar 👇
     @staticmethod
-    def on_progress(d):
-        """Progress callback"""
-        if d["status"] == "finished":
-            print("Done downloading")
-        elif d["status"] == "downloading":
-            print(f"Downloading... progress: {d['_percent_str']}")
-        elif d["status"] == "error":
-            print(f"Error during download: {d['error']}")
+    def wrapper_hook(file_name):
+        def on_progress(d):
+            """Progress callback"""
+            if d["status"] == "finished":
+                print("Done downloading")
+                filename=d['filename']
```

Comment:
wrong variable name `filename`, rename it to `file_name`

Reply:
@Ronald3217 aqui encontre un error. los el nombre la variable no es la misma, la estas definiendo como `filename`, pero la estas llamando como `file_name`. **Renombrarlo a `file_name`**